presentation/diarywrite/src/main/java/com/hilingual/presentation/diarywrite/component/RecommendedTopicDropdown.kt

```diff
@@ -0,0 +1,113 @@
+package com.hilingual.presentation.diarywrite.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.hilingual.core.common.extension.noRippleClickable
+import com.hilingual.core.designsystem.theme.HilingualTheme
+import com.hilingual.presentation.diarywrite.R
+
+@Composable
+internal fun RecommendedTopicDropdown(
+    engTopic: String,
+    korTopic: String,
+    modifier: Modifier = Modifier
+) {
+    var isKor by remember { mutableStateOf(false) }
+    var expanded by remember { mutableStateOf(false) }  // 확장 여부
+
+    Column(
+        modifier = modifier
+            .clip(RoundedCornerShape(8.dp))
+            .background(HilingualTheme.colors.gray100)
+            .fillMaxWidth()
+            .padding(12.dp),
+        verticalArrangement = Arrangement.spacedBy(16.dp)
+    ) {
+        // '오늘의 추천 주제 참고하기' 영역
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            Text(
+                text = "오늘의 추천 주제 참고하기",
+                style = HilingualTheme.typography.bodyM14,
+                color = HilingualTheme.colors.gray700
+            )
+            Icon(
+                modifier = Modifier
+                    .size(24.dp)
+                    .graphicsLayer {
+                        scaleY = if (expanded) -1f else 1f
+                    }
+                    .noRippleClickable { expanded = !expanded },
+                imageVector = ImageVector.vectorResource(R.drawable.ic_arrow_down_20),
+                contentDescription = null,
+                tint = HilingualTheme.colors.gray400
+            )
+        }
+
+        // 드롭다운 활성화되었을 때 보이는 추천 주제 영역
+        if (expanded) {
```

Comment:
p2
이거 `AnimatedVisibility`라는 위젯으로 감싸서 visible 설정해주는 방법도 있어용!
```suggestion
       AnimatedVisibility(
            visible = expanded,
            // 여기에 뭐 애니메이션 코드 설정 가능
        ) {
```
사용을 고려해보셔도 좋을 것 같습니다

Reply:
요정도는 
```
    Column(
        modifier = modifier
            .animateContentSize()
```

위에 요렇게만해도 꽤나 좋습니다

Reply:
사실 저도 `AnimatedVisibility`를 사용해서 구현했었는데, 이때 첨부한 영상처럼 버벅거리는 이슈가 있어서 코드를 엎었었는데요
(+ 기획/디자인 문서에도 애니메이션에 대한 언급은 없어 우선 순위가 낮다고 판단함)

https://github.com/user-attachments/assets/5050915a-5d25-4783-be3a-9e663629ecea

지금 다시 구현해보니까 또 잘 되네요...?? 안스 이슈인가 엉엉🥲
이번에는 민재 오빠가 말해준 `.animateContentSize()` 방식으로 반영해보겠습니다!

Reply:
좋아요~ 앱잼에선 애니메이션은 아주 후순위 입니다 :) 그래도 시도 해보는게 어디에요 푸하하
animateContentSize 이녀석은 Modifier를 적용한 녀석, 지금은 Column이겠네요. 그 녀석의 사이즈가 커지고 작아질때 애니메이션을 적용해서 사이즈를 변화 시켜줍니다. 떄문에 지금은 내부 컴포넌트가 생성됐다가 사라지는거라 부모의 사이즈가 변하는거겠죠? 이럴때 써주면 편합니다